mat/eigen.go

```diff
@@ -305,46 +307,54 @@ func (e *Eigen) complexEigenTo(dst *CDense, d *Dense) {
 	}
 }
 
-// VectorsTo returns the right eigenvectors of the decomposition. VectorsTo
-// will panic if the right eigenvectors were not computed during the factorization,
-// or if the factorization was not successful.
-//
-// The computed eigenvectors are normalized to have Euclidean norm equal to 1
+// VectorsTo returns the right eigenvectors of the decomposition, stored in-place
+// into dst. The computed eigenvectors are normalized to have Euclidean norm equal to 1
 // and largest component real.
-func (e *Eigen) VectorsTo(dst *CDense) *CDense {
+//
+// If dst is empty, VectorsTo will resize dst to be n×n. If dst is
+// non-empty, VectorsTo will panic if dst is not n×n. VectorsTo will also
```

Comment:
"When dst is non-empty, VectorsTo will panic if dst is not n×n."

Reply:
Done.